components/script/dom/bindings/str.rs

```diff
@@ -113,6 +113,74 @@ pub fn is_token(s: &[u8]) -> bool {
     })
 }
 
+/// Returns whether the language is matched, as defined by
+/// [RFC 4647](https://tools.ietf.org/html/rfc4647#section-3.3.2).
+pub fn extended_filtering(tag: &str, range: &str) -> bool {
+    let range_tags: Vec<&str> = range.split(',').collect();
```

Comment:
It's not actually known what the separator is between different language ranges. I suspect it's the empty space character. @SimonSapin do you happen to know?